code/backtracking/partitionsofSet/set_partitions.cpp

```diff
@@ -0,0 +1,56 @@
+#include<stdio.h>
```

Comment:
Kindly remove this file as this has been merged in your previous pull request 👍

Reply:
true. I was hurrying and added them both.